tempest2/src/main/kotlin/app/cash/tempest2/AsyncLogicalDb.kt

```diff
@@ -157,7 +159,7 @@ interface AsyncLogicalDb : AsyncLogicalTable.Factory {
     keys: KeySet,
     consistentReads: Boolean,
     maxPageSize: Int
-  ): Publisher<ItemSet>
+  ): Flow<ItemSet>
```

Comment:
I'm not an expert with coroutines, but this seems like it would be a breaking change. Have you tested this against an app using the `Publisher` interface?